petromiles-frontend/jest.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {

```

Comment:
Esto resolvió un problema de la configuración. Sin embargo creo que tiene que ver con tener la configuración de Jest que se encuentra en dos archivos. Tanto en `jest.config.js`  como en `package.json`

```suggestion
const { defaults } = require("jest-config");

module.exports = {
  ...defaults,
```